### DISCUSSION TITLE
Sberpay/Yookassa (Russia)

### DISCUSSION THREAD

**Discussion by elsegund0** [inspireui/support#5707] [Request new Native Payment]
Please integrate one of those payments, For Russia it's only working variants.

**Comment by ValentinBrill** [comment]
The Russian Federation is disconnected from the Swift international payment system. And electronic payments within the country can only be made by local payment gateways.

The most profitable payment gateway Yookassa from the major bank SberBank:

https://wordpress.org/plugins/yookassa/
https://yookassa.ru/developers/payment-acceptance/integration-scenarios/mobile-sdks/basics

It would be really cool if you implement this payment gateway. You will have a lot of customers from Russia and Belarus, as all brands in the country have switched to e-commerce.

**Comment by Yan-Aleman** [comment]
Hi team. The most profitable solution would be to add a payment gateway.
CloudPayments:
https://wordpress.org/plugins/cloudpayments-gateway-for-woocommerce/

They are available in Kazakhstan: https://cloudpayments.kz/ - And in Russia: https://cloudpayments.ru. 
Support the national currencies of these countries and international payments on better terms than ‘stripe’. They also have the option of split payments for marketplaces. Built-in apple pay and google pay function.
https://developers.cloudpayments.ru/#sdk-dlya-ios
https://developers.cloudpayments.ru/#sdk-dlya-android
https://cloudpayments.ru/integration/cms/wordpress

// Kazakhstan, Russia, Belarus, Armenia, Georgia, Kyrgyzstan, Azerbaijan and for residents of EU countries.

p/s native support WooCommerce Subscriptions

Thanks.
